vue/components/organisms/pickers/pickers.vue

```diff
@@ -438,7 +438,13 @@ export const Pickers = {
             /**
              * Whether the colors picker has a scroll.
              */
-            scrollColor: false
+            scrollColor: false,
+            /**
+             * The currently centered part, material and
+             * color, necessary to maintain the scroll
+             * after a resize.
+             */
+            scrollCenterElements: {}

```

Comment:
Also, split in 3 props: part, material and color. It's always useful to have a hierarchy as flat as possible. Advantages:
- Documentation: you automatically know the structure. With `scrollCenterElements` I have to look at the code to know what its structure is, even with the comment
- Much easier in terms of reactivity

Reply:
I used this strategy because I could do something like this (https://github.com/ripe-tech/ripe-commons-pluginus/blob/d534cc709268be74a6ec940e3154f85f0f541c38/vue/components/organisms/pickers/pickers.vue#L729), which would be simpler. I can switch for a if